build/gulp/tasks/test-projects.tsx

```diff
@@ -9,23 +9,12 @@ import config from '../../../config'
 import * as tmp from 'tmp'
 
 const { paths } = config
-
-const log = msg => {
```

Comment:
now the output is indistinguishable, it is very hard to understand what is going on and immediately find the step necessary:

![image](https://user-images.githubusercontent.com/9024564/49939642-b30cfb00-fedd-11e8-83c5-02d582615ba0.png)

It will significantly complicate debugging scenarios

Reply:
Agree, I restored your logger